src/site/components/events_list.drupal.liquid

```diff
@@ -1,6 +1,9 @@
 <div class="events-listing events-show">
+  {% assign count = 0 %}
   {% for event in fieldOffice.entity.reverseFieldOfficeNode.entities %}
   {% if event.title.length%}
+  {% assign count = count | plus: 1 %}
```

Comment:
I'm not super familiar with variables in `liquid`, but could you do `assign count = count + 1`?

Alternatively: `assign resultsFound = true` if the actual count doesn't matter.

Reply:
I simplified this a bit, @cvalarida

Reply:
I take that back - increment only works in the loop. Doing + 1 doesn't work in liquid - you have to do the `| plus: 1` filter, @cvalarida